src/layouts/screen-size-rules/breakpoints.ts

```diff
@@ -0,0 +1,16 @@
+export const breakpoints = {
+  xs: 0,
+  'small-mobile': 320,
+  sm: 480,
+  'small-mobile-large': 568,
+  md: 768,
+  lg: 1024,
+  'large-tablet': 1112,
+  xl: 1280,
+};
```

Comment:
this breakpoint shoud come from the 
![image](https://github.com/user-attachments/assets/8baf66b5-c21a-4a31-8c97-024a42eed515)
\OpenScience\src\style-library\themes\tokens\tokens.json